packages/api/src/routes/medical/internal-facility.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import status from "http-status";
+import { deleteFacility } from "../../command/medical/facility/delete-facility";
+import { asyncHandler, getCxIdFromQueryOrFail, getFromParamsOrFail } from "../util";
+
+const router = Router();
+
+/** ---------------------------------------------------------------------------
+ * DELETE /internal/facility/:id
+ *
+ * Deletes the facility corresponding to the customer ID and facility id.
+ *
+ * @returns 200 OK.
+ */
+router.delete(
+  "/:id",
+  asyncHandler(async (req: Request, res: Response) => {
```

Comment:
Why delete on internal if we're also adding one on the regular, API-key based route ([link](https://github.com/metriport/metriport/pull/854/files#diff-b40dabd8e74ab70cc8bcacbc63cacb29ec65fe87f2a25eeeeee7a9f4cccd7e6aR115))? Left-over in one of them?

Reply:
Yea leftover ill remove the api key based one as i dont believe we want to introduce this to our cx's yet.